x-pack/plugins/apm/server/routes/fleet/get_apm_package_policy_definition.ts

```diff
@@ -73,6 +72,9 @@ function getApmPackageInputVars({
 }): Record<string, { type: string; value: any }> {
   const overrideValues: Record<string, any> = {
     url: cloudPluginSetup?.apm?.url, // overrides 'apm-server.url' to be the cloud APM host
+    rum_allow_origins: ensureValidMultiText(
+      apmServerSchema[INPUT_VAR_NAME_TO_SCHEMA_PATH.rum_allow_origins]
+    ), // fixes issue where "*" needs to be wrapped in quotes to be parsed as a YAML string
```

Comment:
This is specific to `rum_allow_origins` since it's the only `type: "text", multi: true` input var that accepts wildcard string values (allows all origins). But we could apply this to all similar values if we want to.